### DISCUSSION TITLE
Why does orce.php fetch the post bbcode rather than use the html straight away?

### DISCUSSION THREAD

**Discussion by pufferfish101007** [RedGuy12/perfect-pearls#3] [Q&A] [answered]
In orders/OrCe.php, it fetches the post data, then uses the bbcode even though scratchdb already has the html available. Why does it do this? It would be a lot faster and easier just to use the html and update our css to match scratch's.
If there isn't any particular reason I'll update it and also output it to the page.

Also on a side note, we don't need to do all the parsing, just use response.json().

**Comment by cobaltt7** [comment, marked as answer]
1. Yeah I had some ideas that I decided to not use so we could use html

2. I wrote that before i knew about responce.json() :P
________________________________
